README.rst

```diff
@@ -25,6 +25,8 @@ Once running launch the sample ``python-server`` application::
         -H 'Content-Type: application/json' \
         http://localhost:8080/v2/apps
 
+Then you should be able to use the application in your web browser at http://ping.127.0.0.1.xip.io:8000
+
```

Comment:
We should probably rename this domain since I'm no longer doing a ping here, I'm running the python web server.
Perhaps we should change it to `http://python_server.127.0.0.1.xip.io:8000` ?